purchase_channel_attribute/models/purchase_category.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class PurchaseCategory(models.Model):
+    _name = "purchase.category"
+    _order = "sequence, name"
+
+    name = fields.Char(required=True)
+    sequence = fields.Integer(default=10)
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company",
+        "Company",
+        default=lambda self: self.env["res.company"]._company_default_get(
+            "request.medium"
+        ),
```

Comment:
```suggestion
        default=lambda self: self.env.company,
```
`_company_default_get()` has been deprecated.